tests/Copy-DbaLogin.Tests.ps1

```diff
@@ -35,6 +44,19 @@
 			$login1.Sid | Should be $login2.Sid
 			$login1.Status | Should be $login2.Status
 		}
+		
+		It "Should login with newly created Sql Login (also tests credential login) and gets name" {
```

Comment:
I would drop the "gets name". And then update your test to actually show that it logged in with the `tester`.